mylog.sh

```diff
@@ -100,7 +100,11 @@ shift $((OPTIND-1))
 [ "${1:-}" = "--" ] && shift
 
 if [ "$#" -ne 0 ]; then
-	quickEntry "$@"
+	for var in "$@"
```

Comment:
In bash we have lots of string operators this one might help you keep the code clean:
$*

see also this post https://unix.stackexchange.com/questions/197792/joining-bash-arguments-into-single-string-with-spaces